They look for a pkg-config file that does not exist in our Lua package nor
upstream Lua, but the normal .pc file works just fine.

--

--- a/meson.build
+++ b/meson.build
@@ -108,9 +108,9 @@
 if uopt_lua == 'luajit'
 	lua_opt_dep = [ use_tpt_libs != 'no' ? tpt_libs.get_variable('luajit_dep') : dependency('luajit', static: uopt_static == 'system') ]
 elif uopt_lua == 'lua5.2'
-	lua_opt_dep = [ use_tpt_libs != 'no' ? tpt_libs.get_variable('lua52_dep') : dependency('lua5.2-c++', static: uopt_static == 'system') ]
+	lua_opt_dep = [ use_tpt_libs != 'no' ? tpt_libs.get_variable('lua52_dep') : dependency('lua5.2', static: uopt_static == 'system') ]
 elif uopt_lua == 'lua5.1'
-	lua_opt_dep = [ use_tpt_libs != 'no' ? tpt_libs.get_variable('lua51_dep') : dependency('lua5.1-c++', static: uopt_static == 'system') ]
+	lua_opt_dep = [ use_tpt_libs != 'no' ? tpt_libs.get_variable('lua51_dep') : dependency('lua5.1', static: uopt_static == 'system') ]
 else
 	lua_opt_dep = []
 endif
